test/plugin/test_out_google_cloud.rb

```diff
@@ -87,16 +87,12 @@ def test_partial_success
     d.run
     assert_prometheus_metric_value(
       :stackdriver_successful_requests_count, 1, grpc: false, code: 200)
-    assert_prometheus_metric_value(
-      :stackdriver_failed_requests_count, 0, grpc: false)
```

Comment:
I saw you change the syntax of codes in out_google_cloud.rb. Have you added unit tests for the change? Or test it somewhere making sure it doesn't break anything.

Reply:
These are the actual unit tests. The changes are required for the unit tests to run, because the library changed its API.